spec.html

```diff
@@ -11059,6 +11060,44 @@ <h1>Examples of Automatic Semicolon Insertion</h1>
       <p>In the circumstance that an assignment statement must begin with a left parenthesis, it is a good idea for the programmer to provide an explicit semicolon at the end of the preceding statement rather than to rely on automatic semicolon insertion.</p>
     </emu-clause>
   </emu-clause>
+  <emu-clause id="sec-interesting-cases-of-automatic-semicolon-insertion">
+    <h1>Interesting Cases of Automatic Semicolon Insertion</h1>
+    <em>This section is non-normative.</em>
+    <p>ECMAScript programs can be written in a style with very few semicolons by relying on automatic semicolon insertion. As described above, semicolons are not inserted at every newline, and automatic semicolon insertion can depend on multiple tokens across line terminators.</p>
+
+    <p>As new syntactic features are added to ECMAScript, additional grammar productions could be added that cause lines relying on automatic semicolon insertion preceding them to change grammar productions when parsed. In order to prevent cases where automatic semicolon insertion could change by adding source text outside of a grammar productions existing text boundaries.</p>
```

Comment:
This now just says "In order to prevent <thing>," which isn't a complete sentence.